src/basic/iovec-util.c

```diff
@@ -62,12 +62,12 @@ char* set_iovec_string_field_free(struct iovec *iovec, size_t *n_iovec, const ch
         return x;
 }
 
-void iovec_array_free(struct iovec *iov, size_t n) {
-        if (!iov)
+void iovec_array_free(struct iovec *iovec, size_t n) {
+        if (!iovec)
                 return;
 
         for (size_t i = 0; i < n; i++)
```

Comment:
While at it, maybe use `FOREACH_ARRAY` or `free_many` instead?

Reply:
I switched it to FOREACH_ARRAY.